iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractConnectableXml.java

```diff
@@ -197,13 +197,13 @@ public static void readCurrentLimits(Integer index, CurrentLimitsAdder currentLi
     }
 
     static void writeActivePowerLimits(Integer index, ActivePowerLimits limits, XMLStreamWriter writer, IidmXmlVersion version,
-                                              ExportOptions exportOptions) throws XMLStreamException {
-        writeLoadingLimits(index, limits, writer, version.getNamespaceURI(), version, exportOptions, ACTIVE_POWER_LIMITS);
+                                              boolean valid, ExportOptions exportOptions) throws XMLStreamException {
```

Comment:
Why do you need this `valide` boolean for operational limits ?

Reply:
For the namespace!